src/primitives/index.scss

```diff
@@ -7,3 +7,27 @@
 @import '@primer/primitives/tokens-v2-private/css/tokens/functional/size/size.css';
 @import '@primer/primitives/tokens-v2-private/css/tokens/functional/size/viewport.css';
 @import '@primer/primitives/tokens-v2-private/css/tokens/functional/typography/typography.css';
+
+// Temporary typography vars in rem units variables
+:root {
+  // Heading sizes - mobile
+  // h4-h6 remain the same size on both mobile & desktop
+  --h00-size-mobile: 2.5rem;
+  --h0-size-mobile: 2rem;
+  --h1-size-mobile: 1.625rem;
+  --h2-size-mobile: 1.375rem;
+  --h3-size-mobile: 1.125rem;
+
+  // Heading sizes - desktop
```

Comment:
This is off topic, but was wondering how we could replace:

- `--h00-size: 3rem;`
- `--h2-size: 1.5rem;`

since they don't exist for in the [new typography tokens](https://primer.style/primitives/typography). I guess we could do:

- `--h00-size: 3rem;` -> `2.5rem` (`--primer-text-display-size`)
- `--h2-size: 1.5rem;` -> `1.25rem` (`--primer-text-title-size-medium`)

And to make sure `h2` and `h3` will not look the same, use `--primer-text-subtitle-size` (not bold) for `h3`.

Reply:
On topic for sure! Basically, the new typography tokens were designed to be mapped to a Heading and Text component. We might be able to eventually reconcile our utilities to better map to them, but for now we want to just get these existing values into `rem` units. So this may or may not end up being temporary..